pkg/operator/webhooks.go

```diff
@@ -0,0 +1,59 @@
+package operator
+
+import (
+	"os"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth" // Register the k8s client auth
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+
+	subscriptionsapi_v1alpha1 "github.com/dapr/dapr/pkg/apis/subscriptions/v1alpha1"
+	subscriptionsapi_v2alpha1 "github.com/dapr/dapr/pkg/apis/subscriptions/v2alpha1"
+)
+
+func RunWebhooks(enableLeaderElection bool) {
+	conf, err := ctrl.GetConfig()
+	if err != nil {
+		log.Fatalf("unable to get controller runtime configuration, err: %s", err)
+	}
+
+	mgr, err := ctrl.NewManager(conf, ctrl.Options{
+		Scheme:                 scheme,
+		MetricsBindAddress:     "0",
+		Port:                   19443,
+		HealthProbeBindAddress: "0",
+		LeaderElection:         enableLeaderElection,
+		LeaderElectionID:       "webhooks.dapr.io",
+	})
+	if err != nil {
+		log.Fatal(err, "unable to start webhooks")
+	}
+
+	/*
+		Make sure to set `ENABLE_WEBHOOKS=false` when we run locally.
+	*/
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
This is odd. Maybe DISABLE_WEBHOOKS instead. Also, make it case insensitive.

Reply:
This is the "pattern" as generated by the controller-gen tools. I can certainly make it case insensitive though.